packages/lib/uuid.ts

```diff
@@ -18,4 +19,7 @@ export default {
 	createNanoForInboxEmail: (): string => {
 		return customAlphabet('0123456789abcdefghijklmnopqrstuvwxyz', 8)();
 	},
+	createSecureRandom: (size = 32) => {
+		return nanoidSecure(size);
+	},
```

Comment:
What alphabet does it use in this case? Does it also include "/" or other characters that may be used in URLs?

Reply:
It does not.

> By default, Nano ID uses URL-friendly symbols (A-Za-z0-9_-) and returns an ID with 21 characters (to have a collision probability similar to UUID v4).

https://github.com/ai/nanoid#api